src/cgame/rocket/rocket.cpp

```diff
@@ -525,99 +538,60 @@ Rml::String Rocket_QuakeToRML( const char *in, int parseFlags = 0 )
 		{
 			char c = *token.Begin();
 			const emoticonData_t *emoticon;
-			if ( c == '<' )
-			{
-				if ( span && !spanHasContent )
-				{
-					spanHasContent = true;
-					out.append( spanstr );
-				}
-				out.append( "&lt;" );
-			}
-			else if ( c == '>' )
-			{
-				if ( span && !spanHasContent )
-				{
-					spanHasContent = true;
-					out.append( spanstr );
-				}
-				out.append( "&gt;" );
-			}
-			else if ( c == '&' )
-			{
-				if ( span && !spanHasContent )
-				{
-					spanHasContent = true;
-					out.append( spanstr );
-				}
-				out.append( "&amp;" );
-			}
-			else if ( c == '\n' )
+
+			std::unordered_map<char, std::string> htmlEscapeStrings;
```

Comment:
Declare this as const static and statically intialize the kv?

Reply:
The hash table approach probably has worse performance than if-else and is harder to read in my view.